src/main/java/seedu/address/logic/commands/editcommand/EditIssueCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.commands.editcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+public class EditIssueCommand extends EditCommand {
+
+    /* Possible Fields */
+
+    public static final String ENTITY_TYPE = "issue";
+
+    private EditIssueDescriptor editIssueDescriptor;
+
+    public EditIssueCommand(Id id, EditIssueDescriptor editIssueDescriptor) {
+        super(id);
+        requireNonNull(editIssueDescriptor);
+        /*
+         * Parser should check whether there is anything to edit
+         */
+        this.editIssueDescriptor = editIssueDescriptor;
+    }
+
+    @Override
+    public CommandResult execute(EntityList entityList) throws CommandException {
+        requireNonNull(entityList);
+
+        // IssueList issueList = entityList.getIssueList();
+        Issue issueToEdit = issueList.get(this.id);
+        Issue editedIssue = this.createEditedIssue(issueToEdit, this.editIssueDescriptor);
+
+        /*
+         * TODO: IssueList#update() should check if issue at given ID and editedIssue are not same
+         *  and should check if editedIssue is already contained in the IssueList
+         * TODO: Make EntityListInterface#update() return boolean value?
+         */
+        if (entityList.update(issueToEdit.getId(), editedIssue)) {
```

Comment:
update takes in just the edited entity, it'll check by ID internally before updating. Also, looks like you're expecting the update to return a boolean. 

Should I change the update, delete, and add methods on `EntityList` to return boolean? They currently will throw an Exception and return void.